code/modules/reagents/chemistry/recipes/toxins.dm

```diff
@@ -5,117 +5,136 @@
 	results = list(/datum/reagent/toxin/formaldehyde = 3)
 	required_reagents = list(/datum/reagent/consumable/ethanol = 1, /datum/reagent/oxygen = 1, /datum/reagent/silver = 1)
 	required_temp = 420
+	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_DAMAGING | REACTION_TAG_CHEMICAL | REACTION_TAG_BRUTE | REACTION_TAG_TOXIN
```

Comment:
Formaldehyde doesn't do anything to brute, but it does prevent organ decay after death so I added the organ and other tags.
```suggestion
	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_DAMAGING | REACTION_TAG_CHEMICAL | REACTION_TAG_ORGAN | REACTION_TAG_TOXIN | REACTION_TAG_OTHER
```